pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -464,10 +464,23 @@ func (r *ReconcileLogStorage) generateSecrets(
 	for _, certName := range certs {
 		cert, err := cm.GetCertificate(r.client, certName, common.OperatorNamespace())
 		if err != nil {
-			r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get certificate", err, log)
-			return nil, err
+			// This check is needed because there is a circular dependency between the certs/secrets
+			// used here and with the linseed certs that are created by linseed which this controller
```

Comment:
> linseed certs that are created by linseed 

Linseed itself doesn't create any certificates - which secret is this referring to?